tools/pipelines/build-docs.yml

```diff
@@ -100,27 +100,36 @@ resources:
   pipelines:
   - pipeline: common_definitions
     source: Build - common-definitions
+    branch: $(Build.SourceBranchName)
```

Comment:
Since this pipeline triggers for changes on `main` and any `release/` branches, doesn't that mean that release branches with older contents will clobber the website build? I _think_ this pipeline needs to just run on main.

Reply:
I wonder if we may need a separate pipeline that runs on `main` + `release/*` (and triggers off of the various build pipelines like this one currently does) that bundles and publishes the complete API model artifact.

The `build:docs` pipeline could then just run daily or something and pull the latest published artifacts?